submissions/MarharytaBoichenko/html_css_popup/sass/popUp.scss

```diff
@@ -0,0 +1,263 @@
+body {
+  min-height: 100%;
+  font-family: $mainFont;
+  font-size: 16px;
+  line-height: 1.71;
+  letter-spacing: 0.03em;
+  color: $primaryColor;
+  background-image: url('../popup-icons/ua.jpeg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.list {
+  list-style: none;
+  padding-left: 0;
+  margin: 0;
+}
+
+.visually-hidden {
+  position: absolute;
+  clip: rect(1px 1px 1px 1px);
+  padding: 0;
+  border: 0;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+}
+
+.wrapper {
+  display: grid;
+  grid-template: minmax(100px, auto) 1fr / 1 fr;
+}
+
+.header {
+  padding: 15px 30px;
+  display: grid;
+  grid-template: 1fr / 1fr minmax(100px, auto) minmax(100px, auto);
+  justify-content: end;
+  background-color: transparent;
+  border-bottom: 2px solid rgb(163, 165, 167);
+  column-gap: 30px;
+}
+
+.nav {
+  display: grid;
+  grid-column: 2 / 3;
+}
+
+.nav__list {
+  display: grid;
+  grid-auto-flow: column;
+  align-content: end;
+  gap: 20px;
+}
+
+.nav__link {
+  font-size: 16px;
+  font-weight: 700;
+  color: #5d5c5c;
+}
+
+.nav__link:hover,
+.nav__link:focus {
+  color: #2b2a2a;
+}
+
+.account-block {
+  // position: relative;
+  display: grid;
+  grid-auto-flow: column;
+  align-items: center;
+  gap: 15px;
+}
+
+.account-block__item:not(:last-child) {
+  margin-right: 15px;
+}
+
+.account-block__menu {
+  display: block;
+  width: 35px;
+  height: 35px;
+  background-image: url('../popup-icons/popup-button.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.account-block__checkbox {
+  position: absolute;
+  width: 35px;
+  height: 35px;
+  cursor: pointer;
+  appearance: none;
+  outline: none;
+}
+
+.account-block__checkbox:checked ~ .menu {
+  position: absolute;
+  right: 0;
+  top: 70px;
+  display: block;
+  width: 330px;
+  height: 420px;
+  padding: 25px;
+  overflow-y: auto;
+  background-color: $bcgColor;
+  border-radius: 3px;
+}
+
+.account-block__checkbox:focus ~ .user__menu {
+  border: 2px solid #808080;
+  border-radius: 3px;
+}
+
+.menu {
+  display: none;
+  position: relative;
+}
+
+.menu__list {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr) minmax(40px, auto);
+}
+
+.menu__item {
+  display: grid;
+  justify-items: center;
+}
+
+.menu__item--more {
+  position: relative;
+  display: grid;
+  grid-column: 1 / 4;
+  width: 100%;
+}
+
+.menu__link {
+  width: 100%;
+  font-size: 12px;
```

Comment:
Try to use relative units `rem` and `em`, it's a good practice nowadays. 
You can read articles [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units), [here](https://www.freecodecamp.org/news/what-is-rem-in-css/) or any related to it's info